governance.md

```diff
@@ -0,0 +1,136 @@
+
+Institutions become eligible to become an Institutional Partner by employing individuals who actively contribute to The Project as part of their official duties. To state this another way, the only way for a Partner to influence the project is by actively contributing to the open development of the project, in equal terms to any other member of the community of Contributors and Council Members. Merely using Jupyter/IPython Software or Services in institutional context do not allow an entity to become an Institutional Partner. Financial gifts do not enable an entity to become an Institutional Partner. Once an institution becomes eligible for Institutional Partnership, the Steering Council must nominate and approve the Partnership.
+
+If an existing Institutional Partner no longer has a contributing employee, they will be given a 1 year grace period for remaining employees to begin contributing.
+
+An Institutional Partner is free to pursue funding for their work on The Project through any legal means. This could involve a non-profit organization raising money from private foundations and donors or a for-profit company building proprietary products and services that leverage Project Software and Services. Funding acquired by Institutional Partners to work on The Project is called Institutional Funding. However, no funding obtained by an Institutional Partner can override The Project BDFL and Steering Council. If a Partner has funding to do Jupyter/IPython work and the Council decides to not pursue that work as a project, the Partner is free to pursue it on their own. However in this situation, that part of the Partner’s work will not be under the Jupyter/IPython umbrella and cannot use the Project trademarks in a way that suggests a formal relationship.
+
+To acknowledge institutional contributions there are two level of Institutional Partners, with associated benefits:
+
+**Tier 1** = an institution with at least one Institutional Council Member
+
+-   Acknowledged on the Jupyter/IPython websites, in talks and T-shirts.
+-   Ability to acknowledge their own funding sources on the Jupyter/IPython websites, in talks and T-shirts.
+-   Unlimited participation in the annual Institutional Partners Workshop, held during the (planned) annual Jupyter Project Retreat. This allows the Institutional Partner to invite as many of their own employees and funding sources and collaborators as they want, even if they are not project Contributors or Council Members.
+-   Ability to influence the project through the participation of their Council Member.
+-   Council Members invited to bi-annual Jupyter/IPython Developer Meeting.
+
+Existing **Tier 1** level Institutional Partners:
+
+-   UC Berkeley (Fernando Pérez, Min Ragan-Kelley, Thomas Kluyver)
+-   Cal Poly (Brian Granger)
+-   Rackspace (Kyle Kelley)
+-   Continuum Analytics (Damian Avila)
+
```

Comment:
(Bloomberg) Jason Grout should be here as well...
I guess also Thomas and MinRK's new institutions, but not sure if the association is immediate or there is a "buffer" time...